web/site/js/vis/Node.js

```diff
@@ -287,15 +287,26 @@ export default class Node extends VObject {
             selbox.scale.set(selboxsize / NODE_SELECTION_SCALE, selboxsize / NODE_SELECTION_SCALE);
             selbox.anchor.set(0.5, 0.5);
             this.root.addChildAt(selbox, 0);
-            this._selbox = selbox
+            this._selbox = selbox;
+
+            const rangeCircleSize = this.radioRange;
+            let rangeCircle = new PIXI.Graphics();
+            rangeCircle.beginFill(0x20ff20, 0.2);
```

Comment:
nit: maybe better choose a color of [Material Design](https://material.io/resources/color/#!/?view.left=0&view.right=0)

Reply:
You mean going to this Material Design link and picking a color and copying the number? That is fine, I agree to do it. However, Material Design does not seem to be installed for OTNS so far. All used colors were picked by numbers (#xxyyzz).

Reply:
yes, they were selected from material design color tools too.

Reply:
Thanks for explaining.